.gitignore

```diff
@@ -5,3 +5,4 @@
 Cargo.lock
 **/fixtures/*/target/
 **/fixtures/*/packaged/
+**/test-fixtures/buildpacks/*/target/
```

Comment:
We have quite a few different style of naming for the fixture directories - we should fix this at some point later (in a separate PR)

Reply:
i think this could be fixed if the buildpacks for the tests were copied into a temp folder and then used but, for that to work completely, there would need to be some way of setting the workspace directory also.  right now it defaults to the `libcnb.rs` workspace because that is where the integration tests run.

Reply:
I think you might have misread my comment - that wasn't what I was referring to? :-)

Reply:
yeah, my reply was more around the context of why i needed to add this line (which i was hoping to avoid), not the inconsistencies with fixture directory naming 😅